.travis-ci.sh

```diff
@@ -69,5 +69,12 @@ elif [[ $TASK = 'jshint' ]]; then
   grunt test
 else
   # Otherwise compile and check as normal
-  autoreconf -i && ./configure --enable-rdm-tests --enable-ja-rule && make distcheck DISTCHECK_CONFIGURE_FLAGS='--enable-rdm-tests --enable-ja-rule'
+  autoreconf -i;
```

Comment:
You should keep the ampersands, there's no point running configure if autoreconf failed etc.